production/loki-mixin/dashboards/loki-operational.libsonnet

```diff
@@ -147,12 +153,33 @@ local utils = import 'mixin-utils/utils.libsonnet';
                                local isRowHidden(row) =
                                  std.member(dashboards['loki-operational.json'].hiddenRows, row),
 
+                               local isPanelHidden(panelTitle) =
+                                 std.member(dashboards['loki-operational.json'].hiddenPanels, panelTitle),
+
+                               local replaceCortexGateway(expr, replacement) = if $._config.internal_components then
+                                 expr
+                               else
+                                 std.strReplace(
+                                   expr,
+                                   'job=~"$namespace/cortex-gw(-internal)?"',
+                                   matcherStr(replacement, matcher='job', sep='')
+                                 ),
+
+                               local removeInternalComponents(title, expr) = if (title == 'Queries/Second') then
+                                 replaceCortexGateway(expr, 'queryFrontend')
```

Comment:
This seems complicated. Why do we have to replace anything? Shouldn't it be a configurable variable? Also, is the namespace still cortex-gw in our cloud deployment?

Reply:
We have to remove any panels that use the cortex gateway for people using this w/ OSS. The assumption here is that SSD mode is usually OSS. In the past we've removed/reorganized rows of panels based on SSD vs. microservices, as well as changing job selectors. The problem here is that these panels are part of rows that have a mix of cortex gateway specific panels and panels that are still relevant. Furthermore, we can't just change a job selector in some of these cases as they're using cortex gateway specific metrics.